src/env_properties.h

```diff
@@ -33,6 +33,7 @@
 // Symbols are per-isolate primitives but Environment proxies them
 // for the sake of convenience.
 #define PER_ISOLATE_SYMBOL_PROPERTIES(V)                                       \
+  V(vm_dynamic_import_missing_flag, "vm_dynamic_import_missing_flag")          \
```

Comment:
nit: though the list is not properly sorted, I believe it would be better to add this new item at the end of the list.